spec/helpers/react_helper_spec.rb

```diff
@@ -124,5 +137,25 @@
           with('SearchResultsLayout', hash_including(relatedSearches: [related_search]))
       end
     end
+
+    context 'when an affiliate has news label and news items' do
```

Comment:
* Can you move the let variables related only to this context in here `rss_feed` and `news_items`?
* I think results can be moved as well or just integrated into the `news_items` directly since it's not been used anywhere else
* Add a before block to only to ` allow(search).to receive(:news_items).and_return(news_items)` in this context

Let me know what you think